log/interceptor/interceptor.go

```diff
@@ -82,14 +82,38 @@ func CallLogger(opts ...log.Option) grpc.UnaryServerInterceptor {
 		switch status.Code(err) {
 		case codes.OK:
 			logger.Info("Success.")
-		case codes.Internal:
-			logger.Error("Internal error.")
+		case codes.Canceled:
```

Comment:
This adds support for all gRPC status codes so that logging won't report "User error" for the unrecognized ones. It's in this commit because I used it for testing but I'll make a separate PR for just this change to hopefully commit before this one.